Changes

```diff
@@ -128,11 +128,18 @@ Working version
 
 ### Standard library:
 
+- #13760: Add String.{edit_distance,spellcheck}
+  (Daniel Bünzli, review by wikku, Nicolás Ojeda Bär, Gabriel Scherer and
+   Florian Angeletti)
+
 - #13768: Add Either.get_left and Either.get_right
   (T. Kinsart, review by Nicolás Ojeda Bär and Florian Angeletti)
 
 - #13570, #13794: Format, add an out_width function to Format device for
    approximating unicode width.
+
+- #13570, Format: add an out_width function to Format device for approximating
+   unicode width.
```

Comment:
There is conflict resolution duplication (but I can fix it later).

Reply:
Ah sorry yes. Did it on the way.